src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -72,13 +72,66 @@ def _determine_rgw_addr():
         raise LookupError('No RGW daemon found.')
 
     addr = daemon['addr'].split(':')[0]
-    match = re.search(r'port=(\d+)', daemon['metadata']['frontend_config#0'])
-    if match:
-        port = int(match.group(1))
-    else:
+    port, ssl = _parse_civetweb_config(daemon['metadata']['frontend_config#0'])
+    if port is None:
         raise LookupError('Failed to determine RGW port')
 
-    return addr, port
+    return addr, port, ssl
+
+
+def _parse_civetweb_config(config):
```

Comment:
AFAIK civetweb is optional: http://docs.ceph.com/docs/master/radosgw/frontends/

Reply:
Fixed